lib/index.js

```diff
@@ -31,8 +31,23 @@ export default {
     this.subscriptions.add(atom.config.observe('flow-ide.onlyIfAppropriate', (onlyIfAppropriate) => {
       this.onlyIfAppropriate = onlyIfAppropriate
     }))
+
+    let priority = null
```

Comment:
This should be a property of the default export (`this.priority`, or the original `this.hyperclickPriority`).